pkg/webhook/namespacelabel.go

```diff
@@ -53,6 +53,11 @@ func (l nsSet) Set(s string) error {
 // AddLabelWebhook registers the label webhook server with the manager
 func AddLabelWebhook(mgr manager.Manager, _ *opa.Client, _ *process.Excluder) error {
 	wh := &admission.Webhook{Handler: &namespaceLabelHandler{}}
+	// TODO(https://github.com/open-policy-agent/gatekeeper/issues/661): remove log injection if the race condition in the cited bug is eliminated.
+	// Otherwise we risk having unstable logger names for the webhook.
+	if err := wh.InjectLogger(log); err != nil {
```

Comment:
Looks like this function always returns `nil` https://github.com/open-policy-agent/gatekeeper/blob/0c5e95aa762905d01357927f0f8054bdc91805d6/vendor/sigs.k8s.io/controller-runtime/pkg/webhook/admission/webhook.go#L125-L127

Reply:
Yeah, though that's not something we wrote, and unfortunately we still need to treat it like it could return an error.

Reply:
ok. I guess this way it's future-proof. but also, hopefully the bug will be fixed upstream so we can remove this later.